src/containers/addNew.js

```diff
@@ -0,0 +1,193 @@
+import React, { Component } from 'react';
+import Ratings from 'react-ratings-declarative';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import './form.scss';
+import { addNewBeer } from '../actions';
+
+class AddNew extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      touched: {
+        beerName: false,
+        rating: false,
+        desc: false,
+      },
+      beerName: '',
+      rating: 1,
+      desc: '',
+      file: '',
+      isDisabled: true,
+    };
+  }
```

Comment:
If you move state to class property (outside constructor), you dont have to add constructor function at all